Help/FAQ.rst

```diff
@@ -93,3 +93,33 @@ our master branch. Right now we use 7 of those checks:
 
 - **continuous-integration/appveyor/pr** This one does the same as the one
   above, but for Windows, both 32 and 64bits versions.
+
+How are labels used in issues and pull requests?
+------------------------------------------------
+
+- ``area/`` labels, such as ``area/documenation``, ``area/bears``, ``area/core``, 
+  or ``area/actions``, refer to a particular section in the code base. 
+
+- ``difficulty/`` labels, such as ``difficulty/newcomer``, ``difficulty/low``, 
+  ``difficulty/medium``, or ``difficulty/high``, indicate that an issue is ready 
+  to work on. New contributors start by tackling a newcomer issue and then move on 
+  to low difficulty issues. Experienced contributors focus their attention on more 
+  difficult issues. 
```

Comment:
explain how we label them. if you need help, ask me on public gitter channel, my views: newcomer->mostly typos and easy things that help you get acquainted with the workflow, low->easy things that ask you to change a feature or implement a bear, medium->something that asks you to design something completely usable by coala and that would affect a lot of things, hard is something crucial that might rework a part of coala or add a new cool feature coala would rely on